docs/rfc/rfc-1.adoc

```diff
@@ -215,7 +215,7 @@ Governable Parameters
   will trigger an early sweep on a wallet.
 - `dust_threshold`: The minimum bitcoin deposit amount for the transaction to
   be considered for a sweep.
-- `base_sweeping_fee_max`: The highest amount of BTC that operators can
+- `base_btc_fee_max`: The highest amount of BTC that operators can
   initially propose as a fee for miners in a sweeping transaction.
 - `sweeping_fee_bump_period`: The amount of time we wait to see if a sweeping
```

Comment:
Could we add `btc_fee_max` here as well?

Reply:
428af5bc8ee9c07f2a4883d66c5351fea66a3525